SS14.Launcher/ViewModels/Login/LoginViewModel.cs

```diff
@@ -20,6 +20,7 @@ public class LoginViewModel : BaseLoginViewModel
     [Reactive] public string EditingPassword { get; set; } = "";
 
     [Reactive] public bool IsInputValid { get; private set; }
+    [Reactive] public bool IsPasswordVisible { get; private set; }
```

Comment:
I get that Avalonia doesn't care, but this should probably be public if it's to be set from the view.

```suggestion
    [Reactive] public bool IsPasswordVisible { get; private set; }
```